main/php_globals.h

```diff
@@ -46,6 +46,11 @@ extern ZEND_API struct _php_core_globals core_globals;
 #define TRACK_VARS_FILES	5
 #define TRACK_VARS_REQUEST	6
 
+/* Syslog filters */
+#define FILTER_NONE		0
```

Comment:
Could you prefix it with `PHP_SYSLOG_` (in this case `PHP_SYSLOG_FILTER_NONE`) to prevent possible conflicts with some 3rd party extensions -it's not probaly so likely but it's better to prefix it anyway just to be sure...

Reply:
It would be also better to move the definitions to `php_syslog.h` and include it to `main.c`

Reply:
Done.